diff --git a/makefile_include.mk b/makefile_include.mk
index d466a202..072aa3fd 100644
--- a/makefile_include.mk
+++ b/makefile_include.mk
@@ -76,7 +76,7 @@ endef
 # by giving them as a parameter to make:
 #  make CFLAGS="-I./src/headers/ -DLTC_SOURCE ..." ...
 #
-LTC_CFLAGS += -I./src/headers/ -DLTC_SOURCE -Wall -Wsign-compare -Wshadow
+LTC_CFLAGS += -I./src/headers/ -DLTC_SOURCE -Wall -Wsign-compare -Wshadow -I ../tomsfastmath/src/headers
 
 ifdef OLD_GCC
 LTC_CFLAGS += -W
diff --git a/src/headers/tomcrypt_custom.h b/src/headers/tomcrypt_custom.h
index b25175b8..c9866df7 100644
--- a/src/headers/tomcrypt_custom.h
+++ b/src/headers/tomcrypt_custom.h
@@ -169,7 +169,7 @@
 /* #define LTM_DESC */
 
 /* TomsFastMath */
-/* #define TFM_DESC */
+#define TFM_DESC
 
 /* GNU Multiple Precision Arithmetic Library */
 /* #define GMP_DESC */
